src/parsec_security/parsec_threat_model/threat_model.md

```diff
@@ -40,10 +57,10 @@ Token using the IP's public key.
 **Confidentiality** : if known (the authentication token), an attacker could impersonate a specific
 application and execute operations in their name.
 
-*Not applicable for deployments with no Identity Provider. In a deployment without an Identity
-Provider tenants will still have an identity in the form of an application name, but all entities in
-the system are trusted not to use identities of other tenants and so confidentiality is not
-critical.*
+*Not applicable for deployments (DSPC) and (DRCT). In a deployment without an Identity Provider
+tenants will still have an identity in the form of an application name or user ID, however entities
+in the system are either unable to use a different identity (in deployment (DSPC)) or trusted to
+only use their own identity (in deployment (DRCT)) so confidentiality is not critical.*
```

Comment:
Not sure but maybe it would be clearer to split that in two like:

*Not applicable for deployment DSPC. ....Explainations why...*
*Not application for deployment DRCT. ...Explainations why...*